### PR TITLE
Remove unnecessary `:nounzip`s.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Depends on a new tag which includes https://github.com/Homebrew/brew/pull/4402.

### CODE REVIEW

File: Formula/leiningen.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Leiningen < Formula
   end
 
   resource "jar" do
-    url "https://github.com/technomancy/leiningen/releases/download/2.8.1/leiningen-2.8.1-standalone.zip", :using => :nounzip
```
Comment (ilovezfs) [inline Formula/leiningen.rb]: Why is this one being removed?

File: Formula/zsh-lovers.rb [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-  url "https://deb.grml.org/pool/main/z/zsh-lovers/zsh-lovers_0.9.1_all.deb",
-      :using => :nounzip
+  url "https://deb.grml.org/pool/main/z/zsh-lovers/zsh-lovers_0.9.1_all.deb"
   version "0.9.1"
   sha256 "011b7931a555c77e98aa9cdd16b3c4670c0e0e3b5355e5fd60188885a6678de8"
 
   bottle :unneeded
 
   def install
-    system "tar", "xf", "zsh-lovers_#{version}_all.deb"
```
Comment (ilovezfs) [inline Formula/zsh-lovers.rb]: how is this not needed?

Comment (reitermarkus) [reply Formula/zsh-lovers.rb]: Because a `deb` is automatically detected as `tar`.

Comment (ilovezfs) [reply Formula/zsh-lovers.rb]: ah ok

File: Formula/leiningen.rb (hunk 1 above)
Comment (reitermarkus) [reply Formula/leiningen.rb]: This `.zip` is actually a `.jar`, which is auto-detected.

**Pull request merged**